src/forms/select/Select.tsx

```diff
@@ -72,14 +71,28 @@ export class Select extends React.PureComponent<SelectProps> {
                 onChange={onChange}
                 onBlur={onBlur}
                 className={className}
-                style={{width: error ? "75%" : "100%"}}
+                style={{width: this.getSelectWidth()}}
             >
                 {!showDefaultSelect && <option selected disabled hidden style={{display: "none"}} value="" />}
                 {children}
             </select>
         );
     }
 
+    // Function to calulate width of select tag
```

Comment:
What do you think if we would have 2 classes with 95% and 100% and we could just return a class to use it?
Why? - it will be more simple to find several parameters of width in one place(above/below of the component)

Reply:
@alexander-rassanov I don't mind creating CSS classes for same but this repo uses CSS of Vmware Clarity and we have not created our custom classes so far.

Reply:
@akate1  - I think, we should create custom classes here. We should not use custom styling unless really necessary.  So below method can be updated to return a class instead of direct style.

Reply:
done

Reply:
done